docs/swap.md

```diff
@@ -166,6 +166,9 @@ In Honeycomb you can query the following:
 swap.error exists
 swap.submitted exists
 swap.success exists
+swap.checker.message exists
```

Comment:
I think you should query in a honeycomb board instead of having some documentation here